src/index.html

```diff
@@ -10,7 +10,34 @@
     <title>Search bar airbnb</title>
     <link rel="stylesheet" href="./style.css">
   </head>
-  <body>
-    <h1>Search bar airbnb</h1>
+  <body class="page-content">
+    <div class="form-container page-content__form-container">
+      <form
+        class="form-container__form form form--large"
+        action="/"
```

Comment:
I'd rather block the submit action here since it results in 404. 

`e.preventDefault()`